src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.card {
+  &__stars {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
```

Comment:
The 'justify-content: space-between;' is not needed here. The stars should be placed next to each other without extra space between them.